vnext/Desktop/LegacyWebSocket.cpp

```diff
@@ -30,8 +30,8 @@ using std::string;
 
 using boostecr = boost::system::error_code const&;
 
-namespace facebook {
-namespace react {
+namespace Microsoft {
```

Comment:
I wonder what is our policy regarding the namespaces?

Reply:
We have discussed this in an internal channel.
We will gradually drop `facebook::react`. It was originally used because we intended to upstream this code to Facebook's repository. This is no longer the case.